runtime/frame_descriptors.c

```diff
@@ -68,30 +68,48 @@ static frame_descr * next_frame_descr(frame_descr * d) {
   }
 }
 
+static intnat count_descriptors(link *list) {
+  intnat num_descr = 0;
+  link *lnk;
+  iter_list(list,lnk) {
+    num_descr += *((intnat*) lnk->frametable);
+  }
+  return num_descr;
+}
+
 static int capacity(caml_frame_descrs table) {
   int capacity = table.mask + 1;
   CAMLassert(capacity == 0 || Is_power_of_2(capacity));
   return capacity;
 }
 
+static void fill_hashtable(caml_frame_descrs *table, link *new_frametables) {
+  link *lnk = NULL;
```

Comment:
Isn't code style usually to add a blank line after local variable declaration?
(same for the function above, but this is a nano nitpick considering the size of those functions, feel free to ignore)

Reply:
This does not seem to be a consistent convention across the module, for example (in the trunk version of this file, or in 4.14) `next_frame_descr`, `cons` and `caml_get_frame_descr` do not contain such a blank line. I have the impression that the blank line is used in the functions written in old C style with no local variables, to make it easy to look up the block of all variable declarations in the function, and not used in the functions written in C99 style with new variables declared in the middle of the function body (inside loops, etc.). I vote to leave the style as-is for these two functions.